yds/examples/README.md

```diff
@@ -0,0 +1,20 @@
+# Example of how to send Yandex Data Streams messages by raw HTTP
+
+Sample project for send messages by row HTTP.
```

Comment:
Sample project to send records within HTTP messages.

Reply:
Fixed